go/iot/arduino/protocol.ino

```diff
@@ -0,0 +1,132 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+//
+// This program shows how to send requests of registration and sending data to the raspberry Pi based on the protocols.
+
+#include <ArduinoJson.h>
+
+#define OK 200
+#define FAIL 404
+void setup() {
+  // Initialize Serial port
+  Serial.begin(9600);
+  while (!Serial) continue;
+}
+
+/*
+ * It would first send request of registration to the raspberry Pi.
```

Comment:
I don't think there's "request of registration" any more. All the request will be "request of record".